### PR TITLE
fix(regions): versions

### PR DESCRIPTION
**Pull request by Dav-14**

### CODE REVIEW

**Review by flemzord — APPROVED** [review]

**Review by paul-nicolas — APPROVED** [review]

File: releases/sdks/go/gen.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ generation:
     oAuth2ClientCredentialsEnabled: true
   telemetryEnabled: false
 go:
-  version: v0.0.0
+  version: 0.0.1
   additionalDependencies: {}
```
Comment (Dav-14) [inline releases/sdks/go/gen.yaml:15]: does this need to change ?

File: releases/sdks/go/formance.go [hunk 2]
```diff
@@ -229,9 +229,9 @@ func New(opts ...SDKOption) *Formance {
...
 			Language:          "go",
 			OpenAPIDocVersion: "v0.0.0",
-			SDKVersion:        "v0.0.0",
+			SDKVersion:        "0.0.1",
 			GenVersion:        "2.384.1",
-			UserAgent:         "speakeasy-sdk/go v0.0.0 2.384.1 v0.0.0 github.com/formancehq/formance-sdk-go/v2",
+			UserAgent:         "speakeasy-sdk/go 0.0.1 2.384.1 v0.0.0 github.com/formancehq/formance-sdk-go/v2",
 			ServerDefaults: []map[string]string{
 				{},
 				{
```
Comment (Dav-14) [inline releases/sdks/go/formance.go:237]: all this file

File: releases/sdks/go/.speakeasy/gen.lock [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
   docVersion: v0.0.0
   speakeasyVersion: 1.351.0
   generationVersion: 2.384.1
-  releaseVersion: v0.0.0
-  configChecksum: f2202d55313914a857427cd0c614fbb1
+  releaseVersion: 0.0.1
+  configChecksum: 5a096b3188a1d1996241f123b477f9fb
 features:
   go:
     additionalDependencies: 0.1.0
```
Comment (Dav-14) [inline releases/sdks/go/.speakeasy/gen.lock:12]: and the lock

File: releases/sdks/go/gen.yaml (hunk 1 above)
Comment (Dav-14) [reply releases/sdks/go/gen.yaml]: after the 2nd precommit it pass

File: releases/sdks/go/formance.go (hunk 2 above)
Comment (Dav-14) [reply releases/sdks/go/formance.go]: after the 2nd precommit there is no new changes

File: releases/sdks/go/.speakeasy/gen.lock (hunk 3 above)
Comment (Dav-14) [reply releases/sdks/go/.speakeasy/gen.lock]: outdated now

**Pull request merged**